src/lib/auth-permissions/index.ts

```diff
@@ -0,0 +1,2873 @@
+/**
```

Comment:
This seems to have been accidentally checked in?

Reply:
Ah yes this was research I was doing for the show/hide internal docs component. Removing now, good catch.